src/topology/uniform_space/cauchy.lean

```diff
@@ -330,3 +344,105 @@ instance complete_of_compact {α : Type u} [uniform_space α] [compact_space α]
 lemma compact_of_totally_bounded_is_closed [complete_space α] {s : set α}
   (ht : totally_bounded s) (hc : is_closed s) : compact s :=
 (@compact_iff_totally_bounded_complete α _ s).2 ⟨ht, is_complete_of_is_closed hc⟩
+
+namespace sequentially_complete
+
```

Comment:
Can you add a comment explaining what you are going to do in this section?

Reply:
Done.